src/js/public/solrwayback_index_page.html

```diff
@@ -5,11 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <title><%= htmlWebpackPlugin.options.title %></title>
```

Comment:
This seems like the easy solution... no way to change the variable without going through the woods and back?

Reply:
This variable is overwritten on runtime (e.g. if you step into the about page or just /solrwayback). So I choose not to use the htmlWebpackPlugin at all and replaced it with out generic title. Other than that I have no explanation for what I did :)

Reply:
:+1: